src/Catalyst.Core.Modules.Dfs/WebApi/V0/Controllers/SwamController.cs

```diff
@@ -0,0 +1,197 @@
+    /// </summary>
+    public class SwarmController : IpfsController
+    {
+        /// <summary>
+        ///   Creates a new controller.
+        /// </summary>
+        public SwarmController(ICoreApi dfs) : base(dfs) { }
+
+        /// <summary>
+        ///   Peer addresses.
+        /// </summary>
+        [HttpGet, HttpPost, Route("swarm/addrs")]
+        public AddrsDto PeerAddresses()
+        {
+            var peers = IpfsCore.SwarmApi.GetSwarmKnownPeers(Cancel);
+            var dto = new AddrsDto();
+            foreach (var peer in peers)
+            {
+                dto.Addrs[peer.Id.ToString()] = peer.Addresses
+                   .Select(a => a.ToString())
+                   .ToList();
+            }
+
+            return dto;
+        }
+
+        /// <summary>
+        ///   Connected peers.
+        /// </summary>
+        [HttpGet, HttpPost, Route("swarm/peers")]
+        public async Task<ConnectedPeersDto> ConnectedPeers()
+        {
+            var peers = await IpfsCore.SwarmApi.PeersAsync(Cancel);
+            return new ConnectedPeersDto
+            {
+                Peers = peers.Select(peer => new ConnectedPeerDto(peer)).ToArray()
+            };
+        }
+
+        /// <summary>
+        ///   List the address filters.
+        /// </summary>
+        [HttpGet, HttpPost, Route("swarm/filters")]
+        public async Task<FiltersDto> ListFilters()
+        {
+            var filters = await IpfsCore.SwarmApi.ListAddressFiltersAsync(persist: false, cancel: Cancel);
+            return new FiltersDto
+            {
+                Strings = filters.Select(f => f.ToString()).ToArray()
+            };
+        }
+
+        /// <summary>
+        ///   Add an address filter.
+        /// </summary>
+        /// <param name="arg">
+        ///   A multiaddress.
+        /// </param>
+        [HttpGet, HttpPost, Route("swarm/filters/add")]
+        public async Task<FiltersDto> AddFilter(string arg)
+        {
+            var filter = await IpfsCore.SwarmApi.AddAddressFilterAsync(arg, persist: false, cancel: Cancel);
+            return new FiltersDto
+            {
+                Strings = filter == null ? new string[0] : new[] {filter.ToString()}
+            };
+        }
+
+        /// <summary>
+        ///   Remove an address filter.
+        /// </summary>
+        /// <param name="arg">
+        ///   A multiaddress.
+        /// </param>
+        [HttpGet, HttpPost, Route("swarm/filters/rm")]
+        public async Task<FiltersDto> RemoveFilter(string arg)
+        {
+            var filter = await IpfsCore.SwarmApi.RemoveAddressFilterAsync(arg, persist: false, cancel: Cancel);
+            return new FiltersDto
+            {
+                Strings = filter == null ? new string[0] : new[] {filter.ToString()}
+            };
+        }
+
+        // [HttpGet, HttpPost, Route("swarm/connect")]
+        // public Task Connect(string arg) { return IpfsCore.SwarmApi.ConnectAsync(arg, Cancel); }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)